packages/data-models/appConfig.ts

```diff
@@ -117,6 +117,10 @@ const APP_FOOTER_DEFAULTS: { templateName: string | null } = {
   templateName: null,
 };
 
+const LAYOUT = {
+  main_content_padding: "24px",
+};
```

Comment:
I think the namespace makes sense to me, and agreed would be nice to migrate more layout variables over time from their somewhat verbose alternatives.

I might question how necessary the `main` part of `main_content_padding` is - do we have other specific content that we are assigning padding for which isn't otherwise namespaced? Similarly is `content` indicative (given that content appears in lots of places) - would `page_padding` be more relevant or are we applying this padding in more places than just the template page ion-content? 

I think this is half an authoring preference question, half a dev one

Reply:
I agree that `page_padding` is better, I've renamed the property and updated the PR description (the outstanding [debug content repo PR](https://github.com/IDEMSInternational/app-debug-content/pull/126) is now out of date, but I'll wait until this PR is finalised before updating).

@esmeetewinkel let us know if you have any thoughts about naming this property

Reply:
Happy with `page_padding`